public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,85 @@
+# dvc.api.get_url()
+
+Return the URL to the storage location of the target data (`repo`/`path`).
+
+## Signature
+
+```py
+get_url(path, repo=None, rev=None, remote=None)
+```
+
+## Description
+
+Returns the full URL string (`str` type) to the physical location (in a
```

Comment:
let's put types if needed into singnature. I suggest signature to be more or less in the same format as docstrings ... logic is the same as with CLI - simplify copying things. Also, it'a a good place for strict consistent definitions for params, summary of the return value, not sure about exceptions.

Reply:
So in this case do you mean using a [type hint](https://www.python.org/dev/peps/pep-0484/) like
```py
get_url(path, repo=None, rev=None, remote=None) -> str
```
?

I just copied the signatures from the api module, and they don't specify the type. I think the engineering team discussed this and decided not to use type hints in general.

Reply:
> signature to be more or less in the same format as docstrings...

Sorry, also not getting this idea. We decided no special format for docstrings, just basic English similar to stdlib, but docstrings don't provide a function signature, they explain what it does and mention the arguments etc.

So do you mean to use no signatures in the api ref, basically? And instead copy the docstring over (I would call such a section "definition" instead of "signature".

Reply:
p.s. for now I'm moving this note to the intro linked to https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str, similar to how it's done in `read()`.

Reply:
> not sure about exceptions

Seems like you made your mind in favor of also including them per https://github.com/iterative/dvc.org/pull/908#discussion_r371067026?

Reply:
> Sorry, also not sure I'm understanding this idea. We decided no special format for docstrings, just basic English similar to stdlib

yes, I commented this before that decision was made. Sorry.

Reply:
Np, it's mostly the result of me taking so long to follow up on these PRs... Alright, moving on!